assets/vue/ImageInfo.vue

```diff
@@ -96,7 +119,40 @@
 
          unescapeTag(tag) {
              return he.decode(tag);
+         },
+
+         loadMoreSimiliar() {
+             let url = new URI(`/api/v1/similiar_images/${this.image._id}`);
+             url.addQuery("offset", this.similiarImages.length);
+
+             fetch(url)
+                 .then(resp => {
+                     resp.json().then(jsonResp => {
```

Comment:
Nested promises are ugly, you can instead use:
``` javascript
fetch(url)
    .then(resp => resp.json())
    .then(jsonResp => { /* something something */ });
```

Reply:
Oh. Yeah i guess that's the purpose of using them.